### PR TITLE
Change license to MIT

### PR DESCRIPTION
**Pull request by AregPrograms**
Fixes issue #100 

I know that the decision is not final. I made a PR so when everything is ready @arshadkazmi42 can quickly make the change if wanted/needed.

### CODE REVIEW

**Review by patel-aum — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,27 +1,7 @@
-Copyright (c) 2015, First Issues
-All rights reserved.
+Copyright 2015 First Issues
```
Comment (arshadkazmi42) [inline LICENSE:1]: Can you add `MIT License` at the top. Similar to this [LICENSE](https://github.com/arshadkazmi42/currency-symbols/blob/master/LICENSE)

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Sorry for the late review.

I have added a small change request, Once that is fixed we can get this merged

**Comment by arshadkazmi42** [comment]
@AregPrograms Are you working on this?

**Comment by arshadkazmi42** [comment]
Since there has been no updates closing this.

**Pull request closed without merging**